packages/phase2cli/src/commands/setup.ts

```diff
@@ -351,11 +357,13 @@ const setup = async () => {
         if (confirmation) {
             // Create the bucket.
             const bucketName = getBucketName(ceremonyPrefix)
+            if (!bucketName) showError(GENERIC_ERRORS.GENERIC_NOT_CONFIGURED_PROPERLY, true)
 
             const spinner = customSpinner(`Creating the storage bucket...`, `clock`)
             spinner.start()
 
-            await createS3Bucket(createBucket, bucketName)
+            // @todo should error out
```

Comment:
Why?

Reply:
Did not express it clearly but what I meant was that we should give an appropriate error message should the creation fail

Reply:
You mean inside the Cloud Functions or here for testing?

Reply:
In the actions, if the cf returns false, we could error out - or directly on the cli

Reply:
Oh yes, definitely. This make sense. :+1: In the comment above I was assuming of the testing context but we were on the `phase2cli` package, so nvm again :rofl: